spec/spec_helper.rb

```diff
@@ -96,14 +96,16 @@
   #   # order dependency and want to debug it, you can fix the order by providing
   #   # the seed, which is printed after each run.
   #   #     --seed 1234
-  # config.order = :random
+  config.order = :random
   #
   #   # Seed global randomization in this process using the `--seed` CLI option.
   #   # Setting this allows you to use `--seed` to deterministically reproduce
   #   # test failures related to randomization by passing the same `--seed` value
   #   # as the one that triggered the failure.
-  # Kernel.srand config.seed
+  Kernel.srand config.seed
   config.extend WithModel
+
+  config.bisect_runner = :shell
```

Comment:
What is this for?

Reply:
Default bisect runner is `fork` which tries to run all the tests within a single system process. `shell` is executing each run by invoking `rspec` command in a shell subprocess. The later is quite  a bit slower, but `fork` has a tendency to end up in a Mutex deadlock - and in our case it did pretty much all the time making bisect completely unusable.